crates/store/re_chunk_store/tests/gc.rs

```diff
@@ -387,6 +387,8 @@ fn protected_time_ranges() -> anyhow::Result<()> {
         }
     }
 
+    eprintln!("{store}");
```

Comment:
```suggestion
```

debug vestigue?

Reply:
On purpose -- I like the extra context when things fail.